### ISSUE TITLE
[java] InvalidLogMessageFormat false positive on externally formatted strings

### ISSUE THREAD

**Issue by ctoomey** [pmd/pmd#4172]
**Affects PMD Version:** 6.49.0

**Rule:** [InvalidLogMessageFormat](https://pmd.sourceforge.io/pmd-6.49.0/pmd_rules_java_errorprone.html#invalidlogmessageformat)

**Description:**

Incorrectly flags the following LOGGER line with error "InvalidLogMessageFormat:	Missing arguments, expected 2 arguments but have 0"

**Code Sample demonstrating the issue:**

```java
    String msg = "Got kafka msg, headers=%s, body=%s".formatted(headers, new String(body, UTF_8));
    LOGGER.info(msg);
    return msg;
```

**Expected outcome:**

PMD reports a violation at line 2, but that's wrong. That's a false positive.

**Running PMD through:** *Gradle*

**Comment by ctoomey** [comment]
Thanks @adangel !
